es-ds-docs/pages/molecules/form-msg.vue

```diff
@@ -0,0 +1,184 @@
+<script setup lang="ts">
+const showHeaderError = ref(false);
+const showHeaderInfo = ref(false);
+const showHeaderSuccess = ref(false);
+const showSimpleError = ref(false);
+const showSimpleInfo = ref(false);
+const showSimpleSuccess = ref(false);
+
+const propTableRows = [
+    [
+        'show',
+        'false',
+        '(boolean) Whether to show the form message. This component will respond to changes of this prop, ' +
```

Comment:
I don't think it matters a huge amount in general, but I think it's valuable in this instance because it's otherwise not clear whether you're supposed to pass the string "false"/"true" or a boolean

Reply:
In the docs for `Card` it has String variables surrounded by `''`. We could do that, or update the `Card` docs, I'm also noticing it doesn't use `ds-prop-table`

Reply:
Yeah I can see value in standardizing all the props tables to include types, or else doing that and having strings always have quotes around them.. at the moment it's kind of a free-for-all

Reply:
Exactly 😅 I figure it'll be easier to define a standard now than after all of the components are complete

Reply:
Hmmm yeah you've convinced me, I'll make that update now!